### PR TITLE
OAA-420 public nuget

### PR DESCRIPTION
**Pull request by tlmnrnhrdt**

### CODE REVIEW

File: build-test-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+trigger:
+- main
+
+variables:
+  - group: s3PushInstallers
+  - group: pipeline-tokens
+  - name: configuration
+    value: Release
+
+pool: 'vs17_2'
```
Comment (tim-lyon-arup) [inline build-test-deploy.yml:15]: Do we need to use a self hosted pool, or will `windows-2022` do?

File: build-test-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,166 @@
+# Starter pipeline
```
Comment (tim-lyon-arup) [inline build-test-deploy.yml:1]: Maybe update or delete starter comments

File: build-test-deploy.yml [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+      $(Build.ArtifactStagingDirectory)/**/*.nupkg
+      $(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg
+    assetUploadMode: 'replace'
+    isDraft: true
+    isPreRelease: true
+    addChangeLog: false
+  condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/main'), ne(variables.TAG_EXISTS, 'true'))
+  displayName: 'Publishing assets to Github'
+
+- task: NuGetCommand@2
```
Comment (tim-lyon-arup) [inline build-test-deploy.yml:137]: Is this a repeat of the above pack/release steps? Or something subtly different?

File: .github/workflows/jira-branch-check.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+name: Check Jira key in branch exists and is valid
```
Comment (tim-lyon-arup) [inline .github/workflows/jira-branch-check.yml:1]: Does this check make sense for a public repo? Any outside contributors wouldn't have access the the private Jira board?

File: .github/workflows/jira-pr-check.yml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Check Jira key in PR title exists and is valid
```
Comment (tim-lyon-arup) [inline .github/workflows/jira-pr-check.yml:3]: Similar, is Jira check appropriate?

File: .github/workflows/jira-branch-check.yml (hunk 4 above)
Comment (tlmnrnhrdt) [reply .github/workflows/jira-branch-check.yml:1]: We have those checks also for the other public projects (AdSecGH, ComposGH and GsaGH). I am happy to get rid off them though ;)

File: build-test-deploy.yml (hunk 1 above)
Comment (tlmnrnhrdt) [reply build-test-deploy.yml:15]: Are you referring to making this a github action or do we have pipelines with `windows-2022` available on azure?

File: build-test-deploy.yml (hunk 3 above)
Comment (tlmnrnhrdt) [reply build-test-deploy.yml:137]: yea good spot, was a copy/paste mistake..

**Pull request closed without merging**